app/javascript/packs/commentDropdowns.js

```diff
@@ -26,13 +26,31 @@ const initializeArticlePageDropdowns = () => {
       continue;
     }
 
+    const isProfilePreview = dropdownTrigger.id.includes(
+      'comment-profile-preview-trigger',
+    );
+
     const dropdownContentId = dropdownTrigger.getAttribute('aria-controls');
     const dropdownElement = document.getElementById(dropdownContentId);
 
     if (dropdownElement) {
       const { closeDropdown } = initializeDropdown({
         triggerElementId: dropdownTrigger.id,
         dropdownContentId,
+        onOpen: () => {
+          if (isProfilePreview) {
+            document
```

Comment:
**(refactor, non-blocking)**: 

@aitchiss I noticed that we assigned a variable earlier on:
```
const dropdownElement = document.getElementById(dropdownContentId);
```
Are we able to use the const `dropdownElement` here and in the `onClose` function?